eth/eventhandler/handlers.go

```diff
@@ -263,16 +273,17 @@ func (eh *EventHandler) validatorAddedEventToShare(
 	sharePublicKeys [][]byte,
 	encryptedKeys [][]byte,
 ) (*ssvtypes.SSVShare, *bls.SecretKey, error) {
-	validatorShare := ssvtypes.SSVShare{}
+	validatorShare := &ssvtypes.SSVShare{}
 
 	publicKey, err := ssvtypes.DeserializeBLSPublicKey(event.PublicKey)
 	if err != nil {
-		return nil, nil, &MalformedEventError{
+		return validatorShare, nil, &MalformedEventError{
```

Comment:
why did we do this change? is returning an empty share anything better or different than returning nil?